tests/vcs/git/GitCommitBearTest.py

```diff
@@ -95,11 +95,13 @@ def test_get_metadata(self):
         metadata = GitCommitBear.get_metadata()
         self.assertEqual(
             metadata.name,
-            "<Merged signature of 'run', 'check_shortlog', 'check_body'"
-            ", 'check_issue_reference'>")
+            "<Merged signature of 'run', 'check_revert', 'check_shortlog'"
+            ", 'check_body', 'check_issue_reference'>")
```

Comment:
please don't change existing test unless there's a reason. I don't see any such change in your patch which might affect this test.
Maybe write a new separate test for this?

Reply:
Okay it all cool :+1: you just added `check_revert`